### PR TITLE
Integrate DB

### PR DESCRIPTION
**Pull request by MorrowCameron**
Integrated the DB connector into main fn

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -41,11 +61,13 @@ def main():
             searchCritea = collectDetailedReservationInfo()
 
         elif userSelection == '5':
-            pass
+            revenue = collectRevenueData()
```
Comment (MorrowCameron) [inline main.py:64]: This is part of next commit, let it be lmao

File: utils.py [hunk 2]
```diff
@@ -1,4 +1,9 @@
 def header():
     print("==============================================")
     print("Dijkstra Inn: The Best Inn in Town")
-    print("==============================================\n")
\ No newline at end of file
+    print("==============================================\n")
+
+
+def executeQuery(connector, query):
```
Comment (MorrowCameron) [inline utils.py:7]: util command for executing a query and returning the answer

Comment (MorrowCameron) [reply utils.py:7]: Would recommend wrapping in a try catch if you are working with a query that can fail

**Comment by AkashPraveen** [comment]
LGTM

**Pull request merged**